Core/Utilities/DEncode.py

```diff
@@ -30,6 +30,18 @@
 from pprint import pprint
 
 
+def _ord(char):
```

Comment:
Why not just applying `ord` everywhere since it works in python2 as well ?

Reply:
It doesn't:

Python 2:
```
In [1]: ord("a") == b"a"[0]
Out[1]: False
```
Python 3:
```
In [1]: ord("a") == b"a"[0]
Out[1]: True
```

At first I tried to add both `ord(x)`  and `x` to the `g_d(En|De)codeFunctions` dictionary to work around it becomes a mess. I prefer this solution as it minimises the chance of breaking Python 2.

Reply:
Ah indeed.
Btw, for consistency, should all the 
`g_dDecodeFunctions[_ord("i")] = decodeInt`
be replaced with 
`g_dDecodeFunctions[_ord(b"i")] = decodeInt`
for consistency ?

Reply:
I don't think so as `ord("i")` is saying "what is the ASCII value of this character" whereas `b"i"` is saying this is a byte array with a single value (`115`). You could argue for `b"i"[0]` for consistency (as this is functionally what is done when decoding the incoming data) but `ord` is clearer IMHO.